operator/test/test.sh

```diff
@@ -196,42 +196,44 @@ test_results() {
     echo -n "."
   fi
 
-  # download the API Server certificate locally and configure gRPC.
-  kubectl get secrets tekton-results-tls -n tekton-results --template='{{index .data "tls.crt"}}' | base64 -d >/tmp/results.crt
-  export GRPC_DEFAULT_SSL_ROOTS_FILE_PATH=/tmp/results.crt
-
   RESULT_UID=$(kubectl get "$pipeline_name" -n "$NAMESPACE" -o yaml | yq .metadata.uid)
 
-  # This is required to pass shellcheck due to the single quotes in the GetResult name parameter.
-  QUERY="name: \"$NAMESPACE/results/$RESULT_UID\""
-
   # Proxy the remote service to localhost.
-  timeout 10 kubectl port-forward -n tekton-results service/tekton-results-api-service 50051 >/dev/null &
+  timeout 10 kubectl port-forward -n tekton-results service/tekton-results-api-service 8080 >/dev/null &
   sleep 5
 
   # create a token associated with a service account
   token=$(kubectl create token "$RESULTS_SA" -n "$NAMESPACE")
 
-  RECORD_CMD=(
-    "grpc_cli"
-    "call"
-    "--channel_creds_type=ssl"
-    "--ssl_target=tekton-results-api-service.tekton-results.svc.cluster.local"
-    "--call_creds=access_token=$token"
-    "localhost:50051"
-    "tekton.results.v1alpha2.Results.GetResult"
-    "$QUERY")
-  RECORD_RESULT=$("${RECORD_CMD[@]}" 2>/dev/null)
-  wait
+  fetch_results_using_rest() {
+    # create the query url for curl, $1 is either 'logs' or 'records'
+    QUERY_URL="https://localhost:8080/apis/results.tekton.dev/v1alpha2/parents/$NAMESPACE/results/$RESULT_UID/$1"
 
-  if [[ $RECORD_RESULT == *$RESULT_UID* ]]; then
-    echo "OK"
-  else
-    echo "Failed"
-    echo "[ERROR] Unable to retrieve record $RESULT_UID from pipeline run $pipeline_name" >&2
-    exit 1
-  fi
-  echo
+    QUERY_CMD=(
```

Comment:
That does not look properly formatted. I'd expect the content to be indented, and the closing `)` to be on its own line, not indented.